Detection Rules additions + side nav reorg (#9765)
DataDog/documentation: content/en/security_monitoring/_index.md

---
title: Security and Compliance Monitoring
kind: documentation
---

<div class="alert alert-warning">
Compliance Monitoring is currently available for private beta. Use this <a href="https://docs.google.com/forms/d/e/1FAIpQLScTA913tNGptcAeNNdWlvgECjekYoDVJwR-OkMtZYnJnq-FWw/viewform">form</a> to request access.
</div>

## Overview

{{< img src="security_monitoring/overview_top.png" alt="Datadog Security Monitoring" >}}

> Datadog Security and Compliance Monitoring unify developer, operation, and security teams through one platform. Use a single dashboard to display devops content, business metrics, and security content. Detect threats, like a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration, to your application and infrastructure in real time, and notify your team of security issues by email, slack, Jira, PagerDuty, or a webhook.

Reviewer: ```suggestion
Datadog Security and Compliance Monitoring unify developer, operation, and security teams through one platform. Use a single dashboard to display DevOps content, business metrics, and security content. Detect threats, like a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration, to your application and infrastructure in real time, and notify your team of security issues by email, slack, Jira, PagerDuty, or a webhook.
```

{{< img src="security_monitoring/takeover_ex.png" alt="Slack Example"  style="width:75%;">}}

> Threats are surfaced in Datadog as Security Signals and can be correlated and triaged in the [Security Signals Explorer][1]. Security Signals are generated by Datadog Security Monitoring with [Detection Rules][2]. Detection Rules detect threats across different sources and are available out of the box for immediate use. Any of the provided detection rules can be cloned to change the configuration. A [new rule][3] can also be created from scratch.

Reviewer: ```suggestion
Threats are surfaced in Datadog as Security Signals and can be correlated and triaged in the [Security Signals Explorer][1]. Security Signals are generated by Datadog Security Monitoring with [Detection Rules][2]. Detection Rules detect threats across different sources and are available out of the box for immediate use. You can clone any of the provided detection rules to change the configuration. You can also a [new rule][3] from scratch to fit your specific use case.
```
when I read this the first time I didn't realize it was giving the user options they can take.

{{< img src="security_monitoring/explorer.png" alt="Security Signals Explorer"  >}}

> Enable runtime security and continuous Compliance Monitoring for your containers and hosts. Use **File Integrity Monitoring (FIM)** to watch for changes to key files and directories on hosts or containers in real time, or **Continuous Compliance** to check against your containers and Kubernetes clusters in order to find configuration issues, as defined in the popular CIS compliance benchmarks for Docker and Kubernetes.

Reviewer: ```suggestion
Enable runtime security and continuous Compliance Monitoring for your containers and hosts. Use **File Integrity Monitoring (FIM)** to watch for changes to key files and directories on hosts or containers in real-time, or **Continuous Compliance** to check against your containers and Kubernetes clusters to find configuration issues, as defined in the popular CIS compliance benchmarks for Docker and Kubernetes.
```
